apps/meteor/client/views/room/contextualBar/Info/hooks/useRoomActions.ts

```diff
@@ -8,75 +8,111 @@ import { useRoomHide } from './actions/useRoomHide';
 import { useRoomLeave } from './actions/useRoomLeave';
 import { useRoomMoveToTeam } from './actions/useRoomMoveToTeam';
 
-type RoomActions = {
+type UseRoomActionsOptions = {
 	onClickEnterRoom?: () => void;
 	onClickEdit?: () => void;
+	resetState?: () => void;
 };
 
-export const useRoomActions = (room: IRoom, { onClickEnterRoom, onClickEdit }: RoomActions, resetState?: () => void) => {
+/**
+ *
+ * @param room
+ * @param param1
```

Comment:
```suggestion
 * @param param1
```
what is this?

Reply:
fixing the docs... it should be good now!